docs/blog/october-contributions-showcase.md

```diff
@@ -0,0 +1,165 @@
+
+This is another easy migration we will soon be kicking off across the ecosystem :rocket:
+
+## milliparsec
+
+In the web server space, [milliparsec](https://github.com/tinyhttp/milliparsec) has gained a huge amount of usage as a replacement for `body-parser`.
+
+At only 11kB, this is much smaller than older alternatives, and much faster!
+
+Excellent work by [v1rtl](https://github.com/talentlessguy) on building this and actively working on feedback from the community :heart:
+
+# Contributions / Improvements
+
+## Storybook
+
+## ESLint plugins (and removing older node support!)
+
+A big win for the community this month - removing support for very old Node versions in a few popular ESLint plugins :tada:
+
+For some time now, these plugins have been a source of considerable bloat for those of us who do not need to support such old engines. It is awesome to see the source projects finally dropping these polyfills in new major (breaking) versions.
+
+Just a couple of plugins that are moving ahead on this:
+
+- `eslint-plugin-react` (thanks to [@MichaelDeBoey93](https://x.com/MichaelDeBoey93))
+- `eslint-plugin-jsx-a11y` (thanks to [v1rtl](https://github.com/talentlessguy) and [@MichaelDeBoey93](https://x.com/MichaelDeBoey93))
+- `eslint-plugin-import` (thanks to [@MichaelDeBoey93](https://x.com/MichaelDeBoey93) again)
+
+**None of these have been released yet** but they are well on their way and are being supported by the maintainers.
+
+Do note, the community lead alternatives will certainly live on as they drop even more opinionated dependencies and logic. If you want to reduce your install footprint and get some performance gains, check out the alternatives docs for them here:
+
+- [`eslint-plugin-import`](https://github.com/es-tooling/module-replacements/blob/main/docs/modules/eslint-plugin-import.md)
+- [`eslint-plugin-react`](https://github.com/es-tooling/module-replacements/blob/main/docs/modules/eslint-plugin-react.md)
+
+## strip-ansi
+
+The community had a realisation recently that `strip-ansi` can be replaced very easily by the built-in [stripVTControlCharacters](https://nodejs.org/api/util.html#utilstripvtcontrolcharactersstr)!
+
+Since then, [Namchee](https://github.com/Namchee), [@ari_perkkio](https://x.com/ari_perkkio) and others have contributed a whole raft of PRs migrating away from the library.
+
+This was a great catch and has allowed us to drop the dependency in dozens of projects already.
+
+If you're using Node 16.x and above, you can drop this one too!
+
+## Docusaurus
+
+This one didn't originate from e18e but is still well worth a mention. [Docusaurus](https://github.com/facebook/docusaurus) is currently tackling a pretty huge project of [improving build performance](https://github.com/facebook/docusaurus/issues/10556).
+
+Great to see another large project taking a focus on performance :pray:
+
+They have already shaved off minutes from their build times :fire: very impressive work by the team.
```

Comment:
```suggestion
They have already shaved off minutes from their build times :fire: very impressive work by [@sebastienlorber](https://x.com/sebastienlorber) and the team.
```

I think it was mostly Sebastien working on this, but I'll leave the "team here to be safe.